### ISSUE TITLE
How does one configure the OAuth provider to use for SSO?

### ISSUE THREAD

**Issue by jimklimov** [share-extras/share-oauth#5]
Hello, I've read through the short README and searched the source code and the JAR delivery, but did not find where one can specify the URL of an OAuth(2) server and its services to use for the SSO activity. It is also not clear whether this plugin (alone or with some other Share Extras) can configurably do or not do some extra actions, such as (optional!) creation of user accounts based on confirmation of proper login from the SSO provider, fallback to standard Alfresco (internal) logins, etc.
Basically, we have another corporate SSO-server with OAuth2 capability, and want to authenticate Alfresco to it. This project mentions LinkedIn, Twitter, Yammer as the providers, but does not state if these are the only ones supported (unlike the other project "share-oauth-sso" hardcoded for Google API, but with somewhat better readme/blog publicity) ;)
Thanks in advance,
//Jim Klimov

**Comment by wabson** [comment]
It is not the goal of this project to provide any kind of SSO function and it is in no way connected with the gdepourtales/share-oauth-sso project which you mention. Sorry, I believe you are looking in the wrong place.
